plugin/plugin_test.go

```diff
@@ -237,6 +237,19 @@ func TestContainerdPlugin(t *testing.T) {
 		},
 	})
 
+	Register(&Registration{
```

Comment:
Should we create a separate Pull request for this plugin unit test and keep this PR just for adding integration test?